web/src/components/LandingPageTiles.test.tsx

```diff
@@ -14,26 +14,38 @@ describe("<LandingPageTiles />", () => {
   });
 
   it("routes user to industry selection when the starting a business button is clicked", async () => {
-    render(<LandingPageTiles />);
+    render(<LandingPageTiles isWelcomePage={false} />);
```

Comment:
these new prop changes in the test here can all be removed per JP's comment about making the prop optional, right?